### DISCUSSION TITLE
Navigation paths summary

### DISCUSSION THREAD

**Discussion by dkoprowski** [plausible/analytics#695] [Anything Else]
Hi, is it possible to see how the user navigated through my website? I was looking in docs but didn't find anything. I can only see how many times users visited a specific URL.

What I am interested in is that the user entered URL `A` then moved to `B` and get out of my website on page `C`. 

So this user path was:
`A -> B -> C`

Is this possible right now to see this in plausible?

**Comment by ukutaht** [comment]
The data is techincally there but we don't have a visualization for this yet. More discussion here:  #149
